config/schedule.rb

```diff
@@ -0,0 +1,3 @@
+every 10.minute do
+  runner 'Card.pending_cards_notification'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.